website/content/en/docs/operator-scope.md

```diff
@@ -4,14 +4,6 @@ linkTitle: Operator Scope
 weight: 50
 ---
 
-- [Namespace-scoped operator usage](#namespace-scoped-operator-usage)
-- [Cluster-scoped operator usage](#cluster-scoped-operator-usage)
-  - [Changed required for a cluster-scoped operator](#changed-required-for-a-cluster-scoped-operator)
-  - [Example for cluster-scoped operator](#example-for-cluster-scoped-operator)
-- [CRD scope](#crd-scope)
-  - [CRD cluster-scoped usage](#crd-cluster-scoped-usage)
-  - [Example for changing the CRD scope from namespace to cluster](#example-for-changing-the-crd-scope-from-namespace-to-cluster)
-
```

Comment:
Remove the menu becuase:
1 - It would no longer have the CRD scope
2- The docs has no menu so we should keep the same standard across the project